documentation/src/mkdocs.yml

```diff
@@ -50,6 +83,17 @@ pages:
       - 'Java API Reference': 'javadoc.md'
```

Comment:
What about the RUST and the client bindings? Should we have a list of bindings somewhere?

Reply:
@tkaitchuck has that assignment. Rust Client has an mdbook docs site we can link out to where appropriate, but I've also asked for Rust and Python sections for the Dev Guide.